packages/labs/gen-wrapper-angular/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
```

Comment:
Should this package be public? I got confused because the `publishConfig` option is `public` and on npm the other gen packages are public: https://www.npmjs.com/search?q=%40lit-labs%2Fgen

Reply:
The `@lit-labs/cli` package doesn't support generating angular wrappers yet so this package is still private and won't publish to npm.
https://github.com/lit/lit/blob/85e9bb0e74efccf83d04947c666689aa1cdd8a86/packages/labs/cli/src/lib/generate/generate.ts#L36-L39

Reply:
Awesome!